### PR TITLE
Update yamls

### PR DESCRIPTION
**Pull request by Ancairon**
Will include today's batch of yamls

### CODE REVIEW

File: modules/activemq/metadata.yaml [hunk 1]
```diff
@@ -20,112 +40,109 @@ setup:
...
+            title: Config
+            enabled: false
           description: A basic example configuration.
-          data: |
+          config: |
             jobs:
               - name: local
                 url: http://127.0.0.1:8161
                 webadmin: admin
-        - name: HTTP authentication
```
[the remark this answers is not in the document]
Comment (Ancairon) [reply modules/activemq/metadata.yaml:117]: must have missed it, will add it in, was investigating this myself too

**Comment by ilyam8** [comment]
@Ancairon can you please tell me your plan? 

- The title is "Update yamls".
- PR was ready for review and it was only activemq
- Later another commit was added that changed apache

Is it finished or do you plan to add more? If 2nd, please mark it as draft/not ready.

**Comment by Ancairon** [comment]
True, I will make it draft and open it once I have no more to add, intent is to have all that I do today in it.

**Comment by Ancairon** [comment]
@ilyam8 marking this ready for review, to get these 3 and the other PR about Cassandra reviewed, so I can see if I am getting it right.

File: modules/chrony/metadata.yaml [hunk 2]
```diff
@@ -60,82 +82,115 @@ setup:
 troubleshooting:
   problems:
     list: []
+info_provided_to_referring_integrations:
+  description: ""
+related_resources:
+  integrations:
+    list: []
```
Comment (Ancairon) [inline modules/chrony/metadata.yaml:89]: Probably chrony can have httpcheck as related integration too?

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**